specifications/rats-eat-profile/spec.ocp

```diff
@@ -0,0 +1,297 @@
+important to note that this profile does not extend any CDDL, as it uses CDDL
+already defined by the IETF CoRIM Draft and DICE Concise Evidence.
+
+The evidence is expressed as a CBOR Web Token (CWT). This profile describes the
+list of claims that must be included in the CWT and how they should be used.
+Additionally, the profile outlines how the COSE_Sign1 fields must be populated
+and asserts that the certificate path of the Attestation Key must be included
+in the unsigned section of the COSE_Sign1 header.
+
+<-- TODO: insert diagram here -->
+
+## CWT Claim Set
+
+The CWT claim set is intentionally minimalistic, serving primarily as an
+integrity-protected wrapper for concise evidence.
+
+1. **eat-profile**
+    * This claim is used by the attester to identify the profile. It **MUST** be present and **SHALL** contain the OID assigned to the OCP Profile. **TODO: OCP to assign OID Value**
+2. **eat-issuer**
+    * This claim is optionally used by the attester to bind the EAT to the certificate chain that issued it. If present, **SHALL** match the SUBJECT Common Name of the Attestation Key (AK) Certificate.
+3. **eat-nonce**
+    * This claim is used by the attester to ensure the freshness of the response. It **MUST** be present and **SHALL** be a string or an array of strings. It **SHALL** contain as minimum the nonce value passed by the requester.
+4. **eat-measurements**
+    * This claim is used by the attester to present the target environment claims that verifier will consume for the appraisal policy. It **MUST** be present and **SHALL** encapsulate a “concise-evidence” using the appropriate IANA media type.
+5. **rim-locators**
+    * This claim is used by the attester to point the verifier to the rim repository. If present, **SHALL** be an array of corim-locator-map (as defined by the IETF CoRIM Draft).
+
+The cwt-eat statement is defined as follows:
+
+## Attesters Topology
+
+In certain setups, an attester may need to gather evidence from additional
+devices, including a **Root of Trust (RoT)**. In such scenarios, this attester
+acts as the lead attester, while the other devices serve as sub-attesters.
+
+Ensuring the freshness of evidence collected by the lead attester from the
+sub-attesters is essential. To achieve this, the attester will present a
+secondary nonce. The Lead Attester creates a random value and uses it to
+request evidence from the sub-attesters.
+
+Both the lead attester and the sub-attesters are required to attest to this
+value: it **MUST** be present in both lead attester and sub-attester evidence
+as nonce claim.
+
+A Remote verifier confirms that the evidence from the sub attester has been
+gathered by the lead attester by verifying that one of the eat-nonce in the
+lead attester’s evidence matches the eat-nonce located in the sub attester’s
+evidence. Additionally, the requester is expected to relax time constraints to
+accommodate any latency the lead attester may experience while collecting
```

Comment:
out of curiosity, when would time constraints be a problem? This statement seems redundant on first read. is there some verifier<->attester protocol with latency requirements where this in particular will cause problems?